src/main/config/config_master.h

```diff
@@ -24,7 +24,8 @@ typedef struct master_t {
     uint8_t magic_be;                       // magic number, should be 0xBE
 
     uint8_t mixerMode;
-    uint32_t enabledFeatures;
+    uint32_t enabledFeatures;               // features that are currently enabled (working set)
+    uint32_t desiredFeatures;               // features to be enabled after reboot (feature latch)
```

Comment:
there should be no change to the configuration for this.  it should be able to be done in memory.